dev/docker-compose.yml

```diff
@@ -137,8 +137,8 @@ services:
       - "7777:8080"
     volumes:
       # TODO: authenticators_oidc/test has a dep on this
-      - ../ci/test_suites/authenticators_oidc/keycloak:/scripts
-      - ../ci/test_suites/authenticators_oidc/keycloak/standalone.xml:/opt/jboss/keycloak/standalone/configuration/standalone.xml
+      - ../ci/keycloak:/scripts
+      - ../ci/keycloak/standalone.xml:/opt/jboss/keycloak/standalone/configuration/standalone.xml
```

Comment:
didn't we move this to `ci/oauth/keycloak`?

Reply:
nice catch thanks